### PR TITLE
Added WG00 attenuation curves

### PR DESCRIPTION
**Pull request by dcorre**
The attenuation curves of Witt & Gordon are now included.
There are some tests for WG00 and C00, so far only for testing the inputs (wavelength and tau_V or Av).

The docstrings for both classes are up to date.

There are 2 examples for the documentation in fit_attenuation.rst, showing how to use these 2 classes to fit an extinction curve.

Inputs are now in microns.

### CODE REVIEW

**Comment by dcorre** [comment]
Hello Karl, 

apparently during the building of the documentation scipy is not found when astropy.modeling is called... I do not understand what is the reason. May be you should add scipy to the required packages to install?

On my laptop I did python setup.py install and then python setup.py build_docs and it worked fine. It did not work with python setup.py develop though.

David

**Comment by karllark** [comment]
You can add in scipy as a requirement in setup.cfg.  The line is near the bottom.

**Comment by dcorre** [comment]
I tried to add scipy to setup.cfg as install_requires = astropy, scipy but during the docs building scipy is still not found. It brought a new error with lapack library for one test so I went back to:
install_requires = astropy

I can not figure out how to solve the problem for the documentation. It is building correctly on my laptop.

File: docs/dust_attenuation/fit_attenuation.rst [hunk 1]
```diff
@@ -15,45 +15,145 @@ In this example, we create some artificial curve with the C00 model
...
+    ax.set_title('Example C00 Fit ')
+
+    ax.legend(loc='best')
+    plt.tight_layout()
+    plt.show()
+
+
+
+Example: Use WG00 to fit C00
+=================
```
Comment (karllark) [inline docs/dust_attenuation/fit_attenuation.rst]: This line is too short.  The number of characters in the "====..." line has to be the same as the text above it.

Comment (karllark) [reply docs/dust_attenuation/fit_attenuation.rst]: This issue happens two other times and is the cause of the readthedocs error.

**Comment by karllark** [comment]
Not sure the scipy error.  Have not seen this before and the dust_extinction package uses the same fitter and so should get the same error.  This may be an issue with readthedocs that might be solved now or soon.
Go ahead and fix doc "===..." issues and we'll see if new tests with travis work.

**Comment by karllark** [comment]
BTW, amazing amount of work you've done on this already!  Very cool!  Looking forward to playing around with the new C00 and WG00 models.

**Comment by karllark** [comment]
I found the issue.  Please change the conda dependencies in the .travis.yml file to be:

        - CONDA_DEPENDENCIES='Cython scipy'

That should fix the scipy issue in the build_docs test.

Found by looking at the dust_extinction travis build.  Must have "solved" this problem in the past for that package and forgotten the solution.  Argh.

**Comment by dcorre** [comment]
Thank you it worked!
I just added scipy as we do not use cython so far.

**Comment by karllark** [comment]
Most excellent.  Going to merge now.

**Comment by karllark** [comment]
Addresses issue #4.

**Pull request merged**